frontend/src/features/projects/dispose/ProjectDisposalExemptionSubmitted.tsx

```diff
@@ -29,3 +29,5 @@ export const ProjectDisposalExemptionSubmitted = () => {
     </Container>
   );
 };
+
+export default ProjectDisposalExemptionSubmitted;
```

Comment:
Any reason for the defaults?

Reply:
yes, code splitting requires default exports for lazy loading components.